src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -612,6 +667,50 @@ Status createPacketAndPushIntoGraph<KFSRequest*>(const std::string& name, const
     return StatusCode::OK;
 }
 
+template <typename T>
+static Status createPacketAndPushIntoGraphPython(const std::string& name, const KFSRequest& request, ::mediapipe::CalculatorGraph& graph, PythonBackend * pythonBackend) {
```

Comment:
Serialization has changed alot in this release, please check latest commits

Reply:
Already rebased. I'll check manually and run tests to make sure it's all good.